### PR TITLE
K8s APM Auto-attach updates 

### PR DESCRIPTION
**Pull request by mangulonr**
Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

**Comment by mangulonr** [comment]
Please don't merge this PR until is fully reviewed by the K8s team

File: src/content/docs/kubernetes-pixie/kubernetes-integration/installation/k8s-agent-operator.mdx [hunk 1]
```diff
@@ -72,199 +85,263 @@ helm upgrade --install k8s-agents-operator k8s-agents-operator/k8s-agents-operat
 
 For a complete list of configuration options, see the [README](https://github.com/newrelic/k8s-agents-operator/tree/main/charts/k8s-agents-operator#values) chart.
 
-## Monitored namespaces [#monitored-namespaces]
+## Configure auto-instrumentation
+
+After APM Auto-attach is all set up in your cluster, the next step is just to roll out the configs required to get it operational. 
+That involves having at least one instrumentation Custom Resource (CR) active in the cluster.
+
```
Comment (ramkrishankumarN) [inline src/content/docs/kubernetes-pixie/kubernetes-integration/installation/k8s-agent-operator.mdx:92]: This is confusing me a lot. This is an "operator" in my mind and when we refer this as "APM Auto-attach" it is confusing. We should change this to "After the operator is all set up...".

**Comment by nbaenam** [comment]
Hi @mangulonr, this PR is a draft. When you're ready to publish, reach out our help-documentation Slack channel and we'll get it merged! Thanks!

File: src/content/docs/kubernetes-pixie/kubernetes-integration/installation/k8s-agent-operator.mdx [hunk 2]
```diff
@@ -159,219 +159,266 @@ Both selectors support `matchLabel` and `matchExpressions`.
...
+                ruby
+            </td>
+            <td>
+                `newrelic-ruby-init:latest`
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+These are the versions available for each agent:
```
Comment (mangulonr) [inline src/content/docs/kubernetes-pixie/kubernetes-integration/installation/k8s-agent-operator.mdx:234]: Should we add these versions available info as part of the previous table, column title can be 'Versions available'?

File: src/content/docs/kubernetes-pixie/kubernetes-integration/installation/k8s-agent-operator.mdx [hunk 3]
```diff
@@ -424,42 +472,48 @@ helm search repo k8s-agents-operator
...
+    * [Ruby](/docs/apm/agents/ruby-agent/api-guides/ruby-custom-instrumentation/)
   </Collapser>
 
   <Collapser
     id="read-only-file-system"
     title="Can I install the K8s APM auto-attach if my applications are running on a read-only file system?"
   >
-    No, because the APM agents are injected at application runtime, it needs access to write to the application container's file system.
+
+    No, you cant. The APM agents are injected at application runtime, it needs access to write to the application container's file system.
```
Comment (mangulonr) [inline src/content/docs/kubernetes-pixie/kubernetes-integration/installation/k8s-agent-operator.mdx:498]: Typo ? No you can't

File: src/content/docs/kubernetes-pixie/kubernetes-integration/installation/k8s-agent-operator.mdx (hunk 2 above)
Comment (nbaenam) [reply src/content/docs/kubernetes-pixie/kubernetes-integration/installation/k8s-agent-operator.mdx:234]: Done.

File: src/content/docs/kubernetes-pixie/kubernetes-integration/installation/k8s-agent-operator.mdx (hunk 3 above)
Comment (nbaenam) [reply src/content/docs/kubernetes-pixie/kubernetes-integration/installation/k8s-agent-operator.mdx:508]: Good catch!

**Review by nbaenam — APPROVED** [review]

**Pull request merged**